spec/02-integration/04-admin_api/17-foreign-entity_spec.lua

```diff
@@ -1,6 +1,6 @@
 local helpers = require "spec.helpers"
 local cjson = require "cjson"
-local utils = require "kong.tools.utils"
+local uuid = require "kong.tools.uuid"
```

Comment:
Any reason why these are not just:
```suggestion
local uuid = require("kong.tools.uuid").uuid
```

Reply:
some code may use two apis: `uuid.uuid()` and `uuid.is_valid_uuid()`, so here we localize the module.

Reply:
@chronolaw, being on module level and `utils.` not used in actual code, it feels unneccessary optimization.

Reply:
I would in this case just:
```lua
local uuid = require("kong.tools.uuid").uuid
local is_valid_uuid = require("kong.tools.uuid"). is_valid_uuid
```
and not:
```lua
local utils = require("kong.tools.uuid")
local uuid = uuid.uuid
local is_valid_uuid = uuid. is_valid_uuid
```